### PR TITLE
Feature: Thank you page on booking complete

### LINKED ISSUE #823: Create 'Thank You' screen after payment (Booking Flow)
Create `Thank You` Screen to finish booking flow (after payment)

Designs: https://www.figma.com/file/0pVPMFJnx07fyZnTG2Z6oXC0/Thank-You?node-id=0%3A1

### PR DESCRIPTION
**Pull request by aalises**
May-27-2019 15-40-03

Closes #823

### CODE REVIEW

File: apps/core/scenes/bookingThankYou/ThankYou.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import InformationSection from '../../components/InformationSection/InformationSection';
+
+type Props = {|
+  +navigation: Navigation,
+|};
+
+type travellingForType = 'business' | 'fun';
+
+type State = {|
+  +isTravellingFor: travellingForType,
```
Comment (jvaclavik) [inline apps/core/scenes/bookingThankYou/ThankYou.js]: maybe `TravellingForType`? Uppercase - T

File: apps/core/scenes/bookingThankYou/ThankYou.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      <ContentContainer>
+        <Illustration name="Feedback" style={styles.illustration} />
+        <Text
+          style={styles.marginVerticalSmall}
+          align="center"
+          size="large"
+          weight="bold"
+          type="primary"
+        >
+          Thank you for booking at Kiwi.com
```
Comment (jvaclavik) [inline apps/core/scenes/bookingThankYou/ThankYou.js]: `for booking [what] at Kiwi.com`

Maybe "your flight"?

Comment (aalises) [reply apps/core/scenes/bookingThankYou/ThankYou.js]: Done!

File: apps/core/scenes/bookingThankYou/ThankYou.js (hunk 1 above)
Comment (aalises) [reply apps/core/scenes/bookingThankYou/ThankYou.js]: Done!

**Comment by FilipMessa** [comment]
@jvaclavik After merge don’t forget delete branch please

**Pull request merged**